Carbon/navigation/screens/Login-SignUp/SignUpScreen.js

```diff
@@ -24,14 +27,15 @@ const SignUpScreen = ({navigation}) => {
         <Image source={{ uri: 'https://i.ibb.co/s9Kfh8p/carbon-logo.png'}} style={{width: 200, height: 100}} testID="image"/>
       </View>
       <KeyboardAvoidingView behavior= 'height' style={styles.content}>
+        <UsernameInput testID="usernameInput" />
```

Comment:
Does this save the username to the database? We may need to add a bit to handleSignUp

Reply:
yeah it looks like
```js
<UsernameInput testID="usernameInput" />
```
doesnt handle username and
```js
await signup(username, password, confirm)